lighting/diffuse/lambert.glsl

```diff
@@ -1,4 +1,5 @@
 #include "../../math/const.glsl"
+#include "../shadingData/new.glsl"
```

Comment:
why new.glsl and not `#include "../shadingData.glsl" `

Reply:
Fair point. Fixed.